manager/testdata/new-blueprints/scenario4/plotter.yaml

```diff
@@ -56,6 +56,44 @@ spec:
         bucket: srcbucket
         object: inventory.parq
     dataformat: parquet
+  - assetId: "m4d-notebook-sample/paysim-step2" # the sink of step 2, consumed by step 1
+    cluster: thegreendragon
```

Comment:
I followed what was there before... @simanadler

Reply:
@revit13 @shlomitk1 Originally there was no cluster in the assets.  @revit13 I believe you suggested adding it as an indication of the cluster in which the data resides.  In the steps the cluster is used to determine where the module should run.  Same term different meanings.  Maybe in assets we should call it storageCluster?

Reply:
@simanadler For blueprints it is crucial to know the deployment cluster. But why would we need to know where an asset is located? It can be on some server outside the deployed clusters, for all we know.

Reply:
I agree with @shlomitk1 here. The storage cluster is a nice to have information for the plotter for debugging and may not be in any cluster (Or may be cross regional (e.g. COS)). This information is crucial for the optimizer etc which should be done at this stage.

Reply:
@shlomitk1 and @froesef I agree with you

Reply:
OK will remove the cluster from asset list